### PR TITLE
Add improved env example and changed env variables

### PR DESCRIPTION
**Pull request by Erikyy**

### CODE REVIEW

**Comment by Orzelius** [comment]
Is this supposed to be WIP (draft)?

**Comment by Erikyy** [comment]
> Is this supposed to be WIP (draft)?

Nah, it is quite small task and I just finished it.

**Comment by Orzelius** [comment]
> Nah, it is quite small task and I just finished it.

Ah, okay, then be sure to mention somewhere that it closes #43

File: be/.env.example [hunk 1]
```diff
@@ -1,8 +1,19 @@
...
+DATABASE_USER=postgres 
+
+#example database password
+DATABASE_PASSWORD=postgres
+
+#example database name
+DATABASE_NAME=postgres
+
+#example server port
+SERVER_PORT=3000
```
Comment (Orzelius) [inline be/.env.example:19]: Add eol and change the port (front end already uses 3000)

File: be/.env.example [hunk 2]
```diff
@@ -1,8 +1,19 @@
-Here is example .env file
+#Here is example .env file
 
+#example database ip address
+DATABASE_IP=0.0.0.0
```
Comment (Orzelius) [inline be/.env.example:4]: Maybe change to `127.0.0.1.` so it would immediately work with the docker db setup

**Review by Orzelius — CHANGES_REQUESTED** [review]

**Comment by Orzelius** [comment]
Also your git signature seems off, this makes it so github is confused about your commits and doesn't assign them to your github user. See how the commiter and your github profile seem to be different.
image

You can read the docs and howto [here](https://docs.github.com/en/github/authenticating-to-github/managing-commit-signature-verification)

**Review by Orzelius — DISMISSED** [review]

**Comment by Orzelius** [comment]
wait what a74fe76?

**Comment by Erikyy** [comment]
> wait what [a74fe76](https://github.com/TPT-Loane/TPT-Loane/commit/a74fe7659693b81ab3b37e0cd6021e69e76fbc5a)?

Accidently again, forgot to sign

**Comment by Orzelius** [comment]
Maybe your git and github emails are different or you haven't added one to another?

**Comment by Erikyy** [comment]
> Maybe your git and github emails are different or you haven't added one to another?

I'm quite sure that I use my own email for signatures

**Comment by Erikyy** [comment]
There, turns out that I had wrong email in git config
face-palm

**Comment by Orzelius** [comment]
yay, it's green now
image

**Review by Orzelius — APPROVED** [review]

**Review by Kasparsu — APPROVED** [review]
Yeah this looks like a good .env.example file right there. No gripes what so ever!

**Pull request merged**